account_payment_order_notification/README.rst

```diff
@@ -0,0 +1,123 @@
+==================================
+Account Payment Order Notification
+==================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fbank--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/bank-payment/tree/13.0/account_payment_order_notification
+    :alt: OCA/bank-payment
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/bank-payment-13-0/bank-payment-13-0-account_payment_order_notification
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/173/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds a button on debit / payment orders to send an email to related partners.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Invoicing > Configuration > Payment Modes* and creates a new record as follows:
+    * Name: Test from customers
+    * Payment method: Manual (inbound)
+    * Selectable in Payment Orders: Checked
+    * Link to Bank Account: Fixed
+    * Fixed Bank Journal: Set any existing journal
+
+#. Go to *Invoicing > Customers > Customers* and creates a new record as follows:
+    * Name: Test customer 1
+    * Email: customer1@test.com
+
+#. Go to *Invoicing > Customers > Customers* and creates a new record as follows:
+    * Name: Test customer 2
+    * Email: (empty)
+
+Usage
+=====
+
+#. Go to *Invoicing > Customers > Invoices* and creates a new record as follows:
+    * Customer: Test customer 1
+    * Payment mode: Test from customers
+    * Invoice lines: Any line
+
+#. Repeat the previous step and create an invoice to "Test customer 2".
+#. Go to *Invoicing > Customers > Invoices* select the invoices created and execute the action called "Post entries".
+#. Go to *Invoicing > Customers > Invoices* select the invoices created and execute the action called "Add to Payment/Debit Order" and creates a payment order.
+#. Click on "Confirm Payments" button.
+#. Click on "Send mails" button.
+#. A wizard will be displayed with customers ("Test customer 2" will not have "To send" field checked).
```

Comment:
Instead of this, just warn that any involved partner with no email will appear with the column "To send" not checked.